04-account-kit-manage-permissions/index.ts

```diff
@@ -0,0 +1,65 @@
+// SDK exposes all you need
+import { Chains, PublicKey, Permission } from '@[SOMETHING]kit'
+import { Account } from '@[SOMETHING]kit/account'
+import { Session } from '@[SOMETHING]kit/session'
+
+
+async function main() {
+    const session = Session.login();
+    const account: Account = await session.getAccount();
+
+    // or
+    // const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Manage account Permissions
+
+    // Get a permission
+    const permission: Permission = account.getPermission("active")
+
+    // Get its public keys
+    const publicKeys: PublicKey[] = permission.keys
+
+    // Get its accounts
+    const accounts: Account[] = permission.accounts
+
+    // Add a new permission
+    await account.addPermission({
+        session, // optional, if not defined the active session which is globally stored will be used
```

Comment:
The session passing convention should be as a second param or we will run into conflicts

How about we say that the last param to a call that transacts is always an optional options object that may include a session?

so you would do `await account.addPermission({..}, {session})`

Reply:
Good point 👍